web/src/utils/columns.ts

```diff
@@ -98,6 +97,17 @@ export interface ColumnConfigDef {
   feature?: Feature;
 }
 
+export interface KubeObj {
+  name: string;
+  kind: string;
+  namespace?: string;
+  showNamespace: boolean;
+}
```

Comment:
following kubeObject parser suggestion:
```suggestion
export interface KubeObj {
  name: string;
  kind: string;
  namespace?: string;
  showNamespace?: boolean;
}
```